tests/tableutils.js

```diff
@@ -826,6 +937,21 @@ describe( 'TableUtils', () => {
 					[ '30', '31', '32' ]
 				] ) );
 			} );
+
+			it( 'should move rowspaned cells to a row below removing it\'s row (other cell is overlapping removed row)', () => {
+				setData( model, modelTable( [
+					[ '00', { rowspan: 3, contents: '01' }, '02', '03', '04' ],
+					[ '10', { rowspan: 2, contents: '12' }, '13', '14' ],
+					[ '20', '23', '24' ]
+				] ) );
+
+				tableUtils.removeRows( root.getChild( 0 ), { at: 1 } );
+
+				assertEqualMarkup( getData( model, { withoutSelection: true } ), modelTable( [
+					[ '00', { rowspan: 2, contents: '01' }, '02', '03', '04' ],
+					[ '20', '12', '23', '24' ]
+				] ) );
+			} );
```

Comment:
This test gives strange results in manual test:
![Zrzut ekranu 2020-04-15 o 13 25 16](https://user-images.githubusercontent.com/1232187/79332184-a575e400-7f1c-11ea-8e34-a4d95afcd753.png)

Reply:
See comment below - it is how browsers renders table :man\_shrugging:

Reply:
This is different case (it changes order of cells `12` and `23`).

Reply:
To double check:

![](https://user-images.githubusercontent.com/247363/79343111-491bc000-7f2e-11ea-825c-52decd0d5e56.gif)

WFM but after changes from your other comment.